web/src/app/home.css

```diff
@@ -0,0 +1,3 @@
+.title__wrapper {
+  justify-items: center;
+}
```

Comment:
I don't think you need a separate `home.css` as the root page `/` is built in as `page.tsx` which already has an associated `page.css`. I'd use one or the other.